client/src/app/menu/menu.component.ts

```diff
@@ -74,17 +89,20 @@ export class MenuComponent implements OnInit {
   }
 
   get instanceName () {
-    return this.serverConfig.instance.name
+    return this.serverConfig?.instance.name || ''
```

Comment:
When `serverConfig` could be undefined/null?

Reply:
Before `ngOnInit`. But maybe it's better to set `getTmpConfig()` as initial value for `serverConfig`.

Reply:
I don't think "before `ngOnInit`" exists

Reply:
It does, if calling `this.instanceName` when setting the class properties initial value. But since we're not doing that I can remove this.